bug/bug.go

```diff
@@ -749,3 +749,22 @@ func (bug *Bug) Compile() Snapshot {
 
 	return snap
 }
+
+// EquivalentBug returns true if two bugs are equal
+func EquivalentBug(expected, actual *Bug) bool {
```

Comment:
This should definitely not be a public API of this package, especially as it **change** `actual`.